### ISSUE TITLE
Dice Roller not working for XdY x Z

### ISSUE THREAD

**Issue by mouse0270** [mouse0270/Beyonds-Toolbox#17]
[LtDarien from D&D Beyond](https://www.dndbeyond.com/forums/d-d-beyond-general/general-discussion/10544-d-d-toolbox-chrome-extension?comment=235)
> Having a problem with the Dice Roller, but only for  XdY * Z rolls, such as for gold. You can see many of those rolls on this page: https://www.dndbeyond.com/compendium/rules/dmg/treasure#TreasureTables
> 
> When you click, the Roller pops up but it just says "{0}"

**Comment by kalebhermes** [comment]
I believe this issue is complete now, is that correct?

**Comment by mouse0270** [comment]
This issue is corrected in the new update. I am still testing it but do think I am pushing it live tomorrow.

I will probably leave this open or create a new issue because I really need to update my dice rolling script because the generic one I am using it starting to get kinda patched together

**Comment by kalebhermes** [comment]
Ah it's fixed for me locally because I've pulled your latest code and deployed it haha, that makes sense! It seems to be working for me, with everything I've tested so far!

**Comment by mouse0270** [comment]
Yup, if you are running the most "beta" / GitHub version you will always get my slightly buggy in test version. lol 

I am still working on improving the GitHub and Character Theme Options. This works pretty well but as I said I need to look into building my own dice roller script so it fits my needs more. Right now I am ghetto patching the one I am using.
